File: android-core/src/test/java/com/mparticle/internal/InternalSessionTest.java
package com.mparticle.internal;

import com.mparticle.mock.MockContext;

import org.junit.Assert;
import org.junit.Test;

import java.util.Locale;

public class InternalSessionTest {

    @Test
    public void testSessionIdsAreCapitalized() {
        InternalSession session = new InternalSession();
        session.start(new MockContext());
        String sessionId = session.mSessionID;
        Assert.assertNotEquals(Constants.NO_SESSION_ID, sessionId);
        Assert.assertEquals(sessionId.toUpperCase(Locale.US), sessionId);
    }
}

File: android-core/src/test/kotlin/com/mparticle/internal/InternalSessionTest.kt
package com.mparticle.internal


import com.mparticle.mock.MockContext
import org.junit.Assert
import org.junit.Test

class InternalSessionTest {
    @Test
    fun testSessionIdsAreCapitalized() {
        val session = InternalSession()
        session.start(MockContext())
        val sessionId = session.mSessionID
        Assert.assertNotEquals(Constants.NO_SESSION_ID, sessionId)
        Assert.assertEquals(sessionId.uppercase(), sessionId)
    }
}